spec/feature/remember_me_spec.rb

```diff
@@ -123,6 +137,10 @@ def redo_mfa(the_user: user)
     click_on I18n.t("mfa.phone.code.fields.submit.label")
   end
 
+  def abort_redo_mfa
+    click_on "Back"
```

Comment:
Should we use the locale here?

Reply:
I'm not actually sure where this word comes from, it's not in our locale files

Reply:
ooh interseting, cool will resolve.

Reply:
must be the publishing component?